src/main/java/step3/domain/Round.java

```diff
@@ -0,0 +1,19 @@
+package step3.domain;
+
+import java.util.ArrayList;
+
+public class Round {
+
+    private ArrayList<Integer> results = new ArrayList<>();
+
+    public void start(RacingCars cars) {
+        for (Car car : cars.getCars()) {
+            car.run();
+            results.add(car.getRunDistance());
+        }
+    }
+
+    public ArrayList<Integer> getResults() {
+        return results;
+    }
+}
```

Comment:
```suggestion
public class Round {

    private List<Integer> results = new ArrayList<>();

    public void start(RacingCars cars) {
        for (Car car : cars.getCars()) {
            car.run();
            results.add(car.getRunDistance());
        }
    }

    public List<Integer> getResults() {
        return results;
    }
}
```
ArrayList는 구현체고, List는 인터페이스입니다.
그래서 이렇게 실제로 타입을 주고받을땐 List로 사용하는게 더 유연하게 사용할 수 있답니다!

Reply:
코멘트 감사합니다! 수정 완료했습니다.